src/sentry/utils/avatar.py

```diff
@@ -27,3 +28,49 @@ def get_gravatar_url(email, size=None, default='mm'):
         gravatar_url += "?" + urllib.urlencode(properties)
 
     return gravatar_url
+
+
+LETAR_COLORS = {
+    'blue': '#25A6F7',
+    'green': '#6FBA57',
+    'orange': '#EC5B33',
+    'red': '#E35141',
+    'pink': '#E56AA6',
+    'purple': '#836CC2'
+}
+
+
+def hash_user_identifier(identifier):
+    return sum(map(ord, identifier))
```

Comment:
The code in the colorhash module or what it's called truncates an md5 which distributes better but is also slower.